--- ./libtool	2016-06-14 13:52:04.649573593 -0300
+++ ./libtool	2016-06-14 14:02:06.317655632 -0300
@@ -6809,6 +6809,12 @@
 	  ;;
 	esac # case $deplib

+        if [ ! -f $lib ]; then
+          lib=$lt_sysroot/$lib
+          echo "lib rewrite: $lib"
+        fi
+
+
 	if test "$found" = yes || test -f "$lib"; then :
 	else
 	  func_fatal_error "cannot find the library \`$lib' or unhandled argument \`$deplib'"
@@ -7499,6 +7505,10 @@
 		  ;;
 		esac
 		else
+        	  if [ ! -f $deplib ]; then
+          	    deplib=$lt_sysroot/$deplib
+          	    echo "lib rewrite: $deplib"
+        	  fi
 		  eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $deplib`
 		  test -z "$libdir" && \
 		    func_fatal_error "\`$deplib' is not a valid libtool archive"
@@ -9702,7 +9712,12 @@
 		func_basename "$deplib"
 		name="$func_basename_result"
 		func_resolve_sysroot "$deplib"
-		eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $func_resolve_sysroot_result`
+		deplib=$func_resolve_sysroot_result
+                if [ ! -f $deplib ]; then
+                  deplib=$lt_sysroot/$deplib
+                  echo "lib rewrite: $deplib"
+                fi
+		eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $deplib`
 		test -z "$libdir" && \
 		  func_fatal_error "\`$deplib' is not a valid libtool archive"
 		newdependency_libs+=" ${lt_sysroot:+=}$libdir/$name"
@@ -9728,6 +9743,10 @@
 	      *.la)
 	        func_basename "$lib"
 		name="$func_basename_result"
+                if [ ! -f $lib ]; then
+                  lib=$lt_sysroot/$lib
+                  echo "lib rewrite: $lib"
+                fi
 		eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $lib`
 		test -z "$libdir" && \
 		  func_fatal_error "\`$lib' is not a valid libtool archive"
@@ -9747,6 +9766,10 @@
 		# the library:
 		func_basename "$lib"
 		name="$func_basename_result"
+               if [ ! -f $lib ]; then
+                  lib=$lt_sysroot/$lib
+                  echo "lib rewrite: $lib"
+                fi
 		eval libdir=`${SED} -n -e 's/^libdir=\(.*\)$/\1/p' $lib`
 		test -z "$libdir" && \
 		  func_fatal_error "\`$lib' is not a valid libtool archive"
