--- ./libtool	2016-06-13 17:25:59.176914646 -0300
+++ ./libtool	2016-06-13 17:52:00.205483375 -0300
@@ -3118,7 +3118,7 @@
   case $func_resolve_sysroot_result in
   =*)
     func_stripname '=' '' "$func_resolve_sysroot_result"
-    func_resolve_sysroot_result=$lt_sysroot$func_stripname_result
+    func_resolve_sysroot_result=$lt_sysroot/$func_stripname_result
     ;;
   esac
 }
@@ -8336,6 +8336,11 @@
 	  ;;
 	esac # case $deplib

+        if [ ! -f $lib ]; then
+          lib=$lt_sysroot$lib
+        fi
+        echo "here? $lib"
+
 	$found || test -f "$lib" \
 	  || func_fatal_error "cannot find the library '$lib' or unhandled argument '$deplib'"

@@ -11281,7 +11286,13 @@
 		func_basename "$deplib"
 		name=$func_basename_result
 		func_resolve_sysroot "$deplib"
-		eval libdir=`$SED -n -e 's/^libdir=\(.*\)$/\1/p' $func_resolve_sysroot_result`
+		if [ ! -f $deplib ]; then
+		  deplib=$lt_sysroot$deplib
+		else
+		  deplib=$func_resolve_sysroot_result
+		fi
+		echo "here? $deplib"
+		eval libdir=`$SED -n -e 's/^libdir=\(.*\)$/\1/p' $deplib`
 		test -z "$libdir" && \
 		  func_fatal_error "'$deplib' is not a valid libtool archive"
 		func_append newdependency_libs " ${lt_sysroot:+=}$libdir/$name"
